addon-resizer/nanny/estimator.go

```diff
@@ -56,3 +56,18 @@ func (e LinearEstimator) scaleWithNodes(numNodes uint64) *api.ResourceRequiremen
 		Requests: requests,
 	}
 }
+
+// LogarithmicEstimator estimates the amount of resources in the way that
+// prevents from frequent updates but may end up with larger resource usage
+// than actually needed (though no more than 50%).
+type LogarithmicEstimator struct {
+	linearEst LinearEstimator
+}
+
+func (e LogarithmicEstimator) scaleWithNodes(numNodes uint64) *api.ResourceRequirements {
+	n := uint64(16)
+	for n <= numNodes {
+		n = n * 3 / 2
+	}
+	return e.linearEst.scaleWithNodes(n)
```

Comment:
Clever: I wouldn't want to touch that awful fixed point library either. On the other hand it's weird for LogarithmicEstimator to depend on LinearEstimator. The math of LogarithmicEstimator is dead simple, and this kind of obfuscates it.

It's worth noting that we've taken the speter.net/go/exp/math/dec/inf dependency out of the API. My understanding is that there's some much saner library we can use.